feature-wallet-api/src/main/java/jp/co/soramitsu/feature_wallet_api/domain/interfaces/WalletInteractor.kt

```diff
@@ -30,11 +30,11 @@ interface WalletInteractor {
 
     fun currentAssetFlow(): Flow<Asset>
 
-    fun transactionsFirstPageFlow(pageSize: Int): Flow<List<Transaction>>
+    fun operationsFirstPageFlow(): Flow<List<Operation>>
 
-    suspend fun syncTransactionsFirstPage(pageSize: Int): Result<Unit>
+    suspend fun syncOperationsFirstPage(pageSize: Int): Result<String?>
```

Comment:
It is not clear what method returns from the signature. Probably we should either put docs or create a type alias for return type (like type alias MeaningfulName = String, syncOperationsFirstPage () : Result<MeaningfulName>)